### PR TITLE
daily-digest-2

### PR DESCRIPTION
**Pull request by yanoandri**
TODO: task #1 - give me a skeleton!
TODO: task #2 - I am a legal employee, include me into your database!

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,7 +1,51 @@
...
+	salary          float64  `json:"salary"`
+	domicile        Domicile `json:"domicile"`
+}
 
 func main() {
-	fmt.Printf("Hello, world or 你好，世界 or Καλημέρα κόσμε or こんにちは世界\n")
-}
\ No newline at end of file
+	// TODO: task #1 - give me a skeleton!
+	//missing comma after salary
```
Comment (weisurya) [inline main.go:24]: woaaa - I didn't intend to forget putting that comma :joy: nice catch!

File: main.go [hunk 2]
```diff
@@ -1,7 +1,51 @@
...
+	country   string `json:"country"`
+	is_remote bool   `json:"is_remote"`
+}
+
+type Employee struct {
+	name            string   `json:"name"`
+	entity          string   `json:"entity"`
+	employee_number int      `json:"employee_number"`
+	salary          float64  `json:"salary"`
+	domicile        Domicile `json:"domicile"`
```
Comment (weisurya) [inline main.go:19]: OOT - have you tried what if you forgot defining the json postfix on here?

File: main.go [hunk 3]
```diff
@@ -1,7 +1,51 @@
...
+    `)
+	var employee Employee
+	if err := json.Unmarshal([]byte(data), &employee); err != nil {
+		fmt.Println("Task #1 failed!")
+		return
+	}
+	// TODO: task #2 - I am a legal employee, include me into your database!
+	var database map[string]Employee
+	database = make(map[string]Employee) //use make map into employee
+	database["Golang"] = employee        // assign employee with "Golang" Key hash map
```
Comment (weisurya) [inline main.go:45]: now you learn about simple hashmap as well. Good job!

Now you don't need to use Redis for caching purpose. If possible, you could use Golang on here for fast retrieval

**Review by weisurya — APPROVED** [review]
good job on completing this task! :star_struck:

File: main.go (hunk 2 above)
Comment (yanoandri) [reply main.go:19]: I've tried but it's on assignment 1, i forgot about the json naming then the class doesn't recognized the json attributes inside class

**Pull request closed without merging**